_episodes/03-sharing.md

```diff
@@ -95,6 +97,150 @@ origin  https://github.com/<your_github_username>/hello-world (push)
 ~~~
 {: .output}
 
+## SSH Background and Setup
+We still need to do a little more setup before we can actually connect to this remote repository.
+We need to set up a way for our local computer to authenticate with GitHub so that GitHub recognizes our computer as belonging to the same person who owns the GitHub repository. 
+
+We will use SSH as our authentication method. SSH stands for Secure Shell Protocol.  SSH is a cryptographic network protocol that allows secure communication between computers using an otherwise insecure network.
```

Comment:
```suggestion
We will use SSH as our authentication method. SSH stands for Secure SHell protocol.  SSH is a cryptographic network protocol that allows secure communication between computers using an otherwise insecure network.
```

I found it distracting that the abbreviation of Secure Shell Protocol is not "SSP," and I think this tweak to the cases of a coupl characters helps with this. I do not feel strongly either way - feel free to reject this suggestion if you do not like it.